docs/ref/models/database-functions.txt

```diff
@@ -751,6 +751,29 @@ Usage example::
     >>> print(author.name_lower)
     margaret smith
 
+``Replace``
+~~~~~~~~~~~
+
+.. class:: Replace(expression, string, replacement='', **extra)
```

Comment:
Should the value of `replacement` here reflect the actual code, i.e. `Value('')`?
Also you have `string` here and `text` in the code...

Reply:
Nicely spotted about `string` and `text`. And yes, it's probably better to match the signature in the documentation and the code.